src/wavm/WAVMWasmModule.cpp

```diff
@@ -1494,4 +1592,89 @@ std::unique_ptr<openmp::PlatformThreadPool>& WAVMWasmModule::getOMPPool()
 {
     return OMPPool;
 }
+
+void WAVMWasmModule::printDebugInfo()
+{
```

Comment:
I've added `printDebugInfo` to a few classes now. It's useful for spitting out a dump of information when things fail, as it can be hard to work out what's going on with just the wasm stacktrace (or only a Python stack trace!)